### PR TITLE
33 channels

### PR DESCRIPTION
**Pull request by KingKemar**
#33

added : 
- channels feature
- channel creation
- update container

### CODE REVIEW

File: Django_connected/connected_house/server/forms.py~ [hunk 1]
```diff
@@ -0,0 +1,91 @@
+from django import forms
```
Comment (Thomas-dot-G) [inline Django_connected/connected_house/server/forms.py~]: We actually don't need that file at all, it should not have been indexed as the .gitignore should prevent that

File: Django_connected/connected_house/server/templates/container.html~ [hunk 2]
```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
```
Comment (Thomas-dot-G) [inline Django_connected/connected_house/server/templates/container.html~]: same as above

File: Django_connected/connected_house/server/templates/mychannels.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ {% else %}
+ <label> you don't have any channels, would you like to 
+<a href="/mychannels/new">create</a>
+ one ? </label>
+ {% endif %}
+  
+</table>
+
+  </div>
+{% endblock content %}
```
Comment (Thomas-dot-G) [inline Django_connected/connected_house/server/templates/mychannels.html]: could use an empty end line

File: Django_connected/connected_house/server/templates/newchannel.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            </div>
+        </div>
+       </form>
+  </div>
+</div>
+<script type="text/javascript">
+
+</script>
+
+{% endblock content %}
```
Comment (Thomas-dot-G) [inline Django_connected/connected_house/server/templates/newchannel.html]: same

File: Django_connected/connected_house/website/models.py~ [hunk 5]
```diff
@@ -3,26 +3,44 @@ from __future__ import unicode_literals
 from django.db import models
```
Comment (Thomas-dot-G) [inline Django_connected/connected_house/website/models.py~]: we do not need that file

**Comment by Thomas-dot-G** [comment]
Appart for that, I am ready to merge it (once some files have been deleted)

File: Django_connected/connected_house/server/forms.py~ (hunk 1 above)
Comment (KingKemar) [inline Django_connected/connected_house/server/forms.py~]: fixed

File: Django_connected/connected_house/server/templates/container.html~ (hunk 2 above)
Comment (KingKemar) [inline Django_connected/connected_house/server/templates/container.html~]: fixed

File: Django_connected/connected_house/server/templates/mychannels.html (hunk 3 above)
Comment (KingKemar) [inline Django_connected/connected_house/server/templates/mychannels.html]: fixed

File: Django_connected/connected_house/server/templates/newchannel.html (hunk 4 above)
Comment (KingKemar) [inline Django_connected/connected_house/server/templates/newchannel.html]: fixed

**Comment by KingKemar** [comment]
Fixed issues, you can merge I think.

File: Django_connected/connected_house/server/templates/mychannels.html (hunk 3 above)
Comment (Thomas-dot-G) [inline Django_connected/connected_house/server/templates/mychannels.html]: Actually it is not

Comment (KingKemar) [inline Django_connected/connected_house/server/templates/mychannels.html]: you were talking about form.py~, fixed that though.

**Comment by KingKemar** [comment]
Should be fine now

**Pull request merged**